### ISSUE TITLE
Finnish localization for WaniKani plugin

### ISSUE THREAD

**Issue by thomasnyman** [LipSurf/plugins#15]
A friend of mine is using LipSurf Plus for WaniKani practice. She's created Finnish language synonyms in WaniKani for reviews. However, since LipSurf defaults to English language speech-to-text, the speech recognition doesn't match against her custom synonyms.

Would it be possible to add the option for Finnish language speech recognition to the WaniKani plugin?

Since the WaniKani plugin isn't part of this community plugin repository, I've created a Gist with Finnish localization of the WaniKani review commands (modeled after the Japanese language localization currently in BETA):

https://gist.github.com/thomasnyman/9bb0de3cccc4c1781e855b4b19ea8bf1

**Comment by mikob** [comment]
That's really cool! I'm afraid it won't work unless your friend makes custom synonyms for every item though. Because the speech recognizer can only recognize one language at a time, it won't be able to discern whether she is saying the English meaning, or a Finnish meaning.

**Comment by thomasnyman** [comment]
That's ok in her case. She has created synonyms for every review item up until her current level.

**Comment by mikob** [comment]
Wow, OK, I think I'll be able to do it then. Perhaps on the next release.

**Comment by thomasnyman** [comment]
Thanks, that would be greatly appreciated!

**Comment by thomasnyman** [comment]
Any chance the Finnish localization for the WaniKani might make it to a future version of LipSurf?

I would be happy to improve on the suggested changes if you have any feedback.

**Comment by mikob** [comment]
@thomasnyman I have the custom plugin for you that will work with Finnish ready. Can you let me know what email you're subscribed with and I will send it there? 

Here's how to install it:
https://help.lipsurf.com/configuration/custom-plugins.html

To change your base lang in WaniKani requires LipSurf v3.7.0 which isn't released yet, so in the meantime you will need to go to the LipSurf options page (say "options") and open up the console (chrome developer tools console) and paste the following:

```
chrome.storage.local.get('pluginPreferences', data => {
data.pluginPreferences.WaniKani.settings['Base Language'] = 'fi';
chrome.storage.local.set({pluginPreferences: data.pluginPreferences}, () => console.log('done'));
});
```

After that, it should work in Finnish. Let me know if you want to add more commands, eg. close help, move help. don't auto open help etc (found in the LipSurf plugin)

**Comment by mikob** [comment]
This has been implemented in #v3.7.0.
